README.md

```diff
@@ -25,27 +25,31 @@ Installation
 
 ###Prerequisites
 
-####Fedora 23
+####Fedora 25
```

Comment:
Maybe we should just remove the version here so this line doesn't become obsolete every 6 months :-)